### ISSUE TITLE
(mobile) Report time/date

### ISSUE THREAD

**Issue by kampf** [ccnz/Ushahidi_Web#74]
Suggest adding a CR after the report time/date on dev mobile.

This would increase the minimum report footprint to three lines, however, it would also make report titles, time/date, and location consistent.  Present formatting puts time/date immediately post-title, therefore making it inconsistently placed.

**Comment by mjollnir** [comment]
Yeah I wasn't sure about that. I actually moved it on purpose to try and increase the size we could have for the map.

**Comment by kampf** [comment]
In this case I'm referring to the report list, not the actual report w/map.

In the report itself the time/date looks well placed.

**Comment by hayatoc** [comment]
The mobile has been updated '14:30 Mar 02(Updated 14:35 Mar 02)'
Don't need it repeated (the updated date isn't on www version)

**Comment by mjollnir** [comment]
The reason we put that there, was that when you ask to sort by date, it actually sorts by modified date , and the ordering looks wrong because it's incident date that is shown.  It could be that the best way to deal with it is to order by incident date instead.
